config/locales/en.yml

```diff
@@ -242,7 +242,7 @@
       firms: firms
 
     result:
-      adviser: Adviser
+      adviser: Firm has adviser
```

Comment:
I think this should be just `has an adviser` as it's contextual to a firm, plus `Firm has adviser` is not even cromulent.